packages/a/argh/xmake.lua

```diff
@@ -0,0 +1,48 @@
+package("argh")
+    set_homepage("https://github.com/adishavit/argh")
+    set_description("Argh! A minimalist argument handler.")
+    set_license("BSD-3-Clause")
+
+    set_urls("https://github.com/adishavit/argh.git")
+    add_deps("cmake")
+
+    on_install(function (package)
+        import("package.tools.cmake").install(package)
```

Comment:
add configs to disable test and examples

BUILD_TESTS=OFF, BUILD_EXAMPLES=OFF

or only use `os.cp("args.h", package:installdir("include"))`

Reply:
It's nice to get rid of cmake